File: GTTS/INL.VettingService.Models/GetPostVettingConfiguration_Result.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.VettingService.Models
{
    public class GetPostVettingConfiguration_Result : IGetPostVettingConfiguration_Result
    {
        public int PostID { get; set; }
        public int MaxBatchSize { get; set; }
        public int LeahyBatchLeadTime { get; set; }
        public int CourtesyBatchLeadTime { get; set; }
        public int LeahyBatchExpirationIntervalMonths { get; set; }
        public int CourtesyBatchExpirationIntervalMonths { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/get-post-vetting-configuration_result.ts


import { IGetPostVettingConfiguration_Result } from './iget-post-vetting-configuration_result';

export class GetPostVettingConfiguration_Result implements IGetPostVettingConfiguration_Result {
  
	public PostID: number = 0;
	public MaxBatchSize: number = 0;
	public LeahyBatchLeadTime: number = 0;
	public CourtesyBatchLeadTime: number = 0;
	public LeahyBatchExpirationIntervalMonths: number = 0;
	public CourtesyBatchExpirationIntervalMonths: number = 0;
  
}


